contrib/bdii/htcondor-ce-provider

```diff
@@ -100,6 +101,8 @@ GLUE2PolicyRule: vo:{vo}
 GLUE2MappingPolicyShareForeignKey: {shareid}
 """
 
+def handler(signum, frame):
+    raise Exception("TimeoutError")
```

Comment:
Could you raise `RuntimeError` or better yet, create a `TimeoutException` class?

Reply:
Add will add a TimeoutError Exception class